tripal_chado/tests/src/Kernel/Services/TripalEntityLookupServiceTest.php

```diff
@@ -1,16 +1,19 @@
 <?php
 
-namespace Drupal\Tests\tripal\Kernel;
+namespace Drupal\Tests\tripal\Kernel\Services;
 
 use Drupal\Tests\tripal_chado\Kernel\ChadoTestKernelBase;
 use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\RunTestsInSeparateProcesses;
 
 /**
  * Tests the entity lookup service for chado-based content types.
  *
  * @group TripalEntityLookup
  */
-#[Group('TripalEntityLookup')]
+#[Group('tripal-content')]
+#[Group('service')]
```

Comment:
Proposing "service-collection" because of only three "service" groups as noted elsewhere. Or maybe "service-lookup" would be better?
```suggestion
#[Group('service-collection')]
```

Reply:
Addressed in 45b53ab60a47d607c134956bbd17ee5c31a74dfe. Ended up going with service-entity-lookup :-). The service-collection is specific to the config entity collections that we use to import terms, content types, fields.